Projects/Application/Sources/Flow/MainFlow/FeedScreen/FeedViewController.swift

```diff
@@ -9,16 +9,18 @@ import RxSwift
 import UIKit
 import Utilities
 
-final class FeedViewController: BaseViewController<FeedViewReactor, FeedView> {
+final class FeedViewController: BaseViewController<FeedViewReactor, FeedView>, UITableViewDelegate {
     override func bind(reactor: Reactor) {
         logger.debug("Binding reactor for FeedViewController")
 
         rx
-            .methodInvoked(#selector(viewDidLoad))
-            .map { _ in Reactor.Action.viewDidLoad }
+            .methodInvoked(#selector(viewDidAppear))
```

Comment:
For my understanding: what is the rationale behind reacting to `viewDidAppear` instead of `viewDidLoad`?

Reply:
viewDidLoad was too fast. For now, it is not important what is used. We can improve it later if need.